window-copy.c

```diff
@@ -2694,6 +2714,18 @@ static const struct {
 	  .clear = WINDOW_COPY_CMD_CLEAR_ALWAYS,
 	  .f = window_copy_cmd_jump_to_mark
 	},
+	{ .command = "next-prompt",
+	  .minargs = 0,
+	  .maxargs = 0,
+	  .clear = WINDOW_COPY_CMD_CLEAR_ALWAYS,
+	  .f = window_copy_cmd_next_prompt
+	},
+	{ .command = "previous-prompt",
+	  .minargs = 0,
+	  .maxargs = 0,
+	  .clear = WINDOW_COPY_CMD_CLEAR_ALWAYS,
+	  .f = window_copy_cmd_previous_prompt
+	},
```

Comment:
> I don't like having to specify an argument - or + to go forward or backwards, can you split into two commands jump-to-prompt-forward and jump-to-prompt-reverse like the other commands?

I saw some `previous-*` and `next-*` commands. So added `previous-prompt` and `next-prompt` matching those.

Let me know if you want to change those to `jump-to-prompt-{forward,backward/reverse}` instead.